frontend/packages/console-app/src/components/network-policies/create-network-policy.tsx

```diff
@@ -8,15 +8,16 @@ import './_create-network-policy.scss';
 
 export const CreateNetworkPolicy: React.FunctionComponent<{}> = () => {
```

Comment:
nit: Other parts of the console use the shorthand `React.FC`. Not important to change for this PR, but it would be good to stay consistent.